apps/blockchain/test/blockchain_test.exs

```diff
@@ -1,4 +1,1486 @@
 defmodule BlockchainTest do
   use ExUnit.Case
+  use EthCommonTest.Harness
+
+  alias Blockchain.{Blocktree, Account, Transaction}
+  alias MerklePatriciaTree.Trie
+  alias Blockchain.Account.Storage
+  alias Block.Header
+
   doctest Blockchain
+
+  @ethereum_common_tests_path "/../../ethereum_common_tests/BlockchainTests/"
+  @tests [
+    "./GeneralStateTests/stAttackTest/ContractCreationSpam_d0g0v0.json",
```

Comment:
How are we deciding which tests to include here vs doing something like we've done in `blockchain/transaction_test.exs`?

Reply:
We're including all tests. Do you suggest specifying only directories? I tried to make it similar to `state_tests.exs`

Reply:
Oh, I see. Yeah `blockchain/state_tests.exs` is also a good example. I don't know if I have a suggestion. Mine was just a question to try to understand which tests we're putting in which files (since I haven't been working on tests, I'm not sure which approach you're taking). 

For example, us this file (`test/blochain_test.exs`) supposed to include _only_ `GeneralStateTests`? Or will all the tests that we are currently testing in places like `state_test.exs` eventually be moved here? If this doesn't make sense, I can also ask on slack or during standup. I'm not sure I'm writing the most understandable question.

Reply:
The idea was to include all tests from `BlockchainTests' directory from ethereum common tests. Maybe there is a confusion because blockchain tests also have general state tests group which tests adding blocks. 

From Ethereum tests' site:
```
Of special notice is the /BlockchainTests/GeneralStateTests folder within the blockchain tests folder structure, which contains a copy of the General State Tests but executes them within the logic of the blockchain tests.
```

Reply:
Oh yeah, that was confusing me. That clarifies things a bit. 

So eventually, we would remove `blockchain/state_test.exs` and `blockchain/transaction_test.exs` because those tests would be here, right?

Reply:
![im)](https://i.imgur.com/YcJImlN.png)
`trantransaction_test.exs` tests files from the `TransactionTests` directory
`state_test.exs` tests files from the `GeneralStateTests` directory
`blockchain_test.exs` tests files from the `BlockchainTests` directory

We can move all json test to separate files or separate directory.

Reply:
Ohh, I see. Sorry. I saw `Blockchain/TransitionTests/` and read "TransactionTests" instead. Yeah, what you're saying makes a lot more sense now. Thanks for explaning!